src/person.rb

```diff
@@ -12,6 +12,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Also, here related to rentals, you should have a method to add rentals to a person as well in order to have the associations set in the correct way, so it may look something like this:
```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
So please make sure to work on this issue.

Reply:
Thank you very much. Added this and added extra test cases for these methods.